### DISCUSSION TITLE
Change auto-uncoop system

### DISCUSSION THREAD

**Discussion by WarnDa** [BG-Software-LLC/SuperiorSkyblock2#368] [Implemented Feature Requests] [answered]
Currently when you coop someone, he is UN-COOP when HE disconnects from the server, but when YOU disconnect, it doesn't un-coop him, 
So you can get "bad surprise" with someone you have cooped and he steals some things when you disconnect.

So an option to change this things would be nice !

Also add a command to coop forever someone would be nice :)
(like '/is coop <player> permanent' )

**Comment by OmerBenGera** [comment, marked as answer]
Implemented in https://github.com/BG-Software-LLC/SuperiorSkyblock2/tree/feature/auto-uncoop
Will be merged to the official builds soon :)
